runtimes/runtimes/operational-telemetry/aws-spans-exporter.ts

```diff
@@ -29,6 +29,17 @@ export class AwsSpanExporter implements SpanExporter {
             resultCallback({ code: ExportResultCode.FAILED })
             return
         }
+        if (this.telemetryService.getTelemetryStatus() === TelemetryStatus.Pending) {
+            // add to queue
+            resultCallback({ code: ExportResultCode.SUCCESS })
+            return
+        }
+        if (this.telemetryService.getTelemetryStatus() === TelemetryStatus.Disabled) {
```

Comment:
Why is it needed - isn't it enough to shutdown `NodeSDK`?

Reply:
I've considered the corner cases in this scenario. When `sdk.shutdown()` is called, it triggers `metricReader.shutdown()`, which in turn initiates a flush of collected metrics (https://github.com/open-telemetry/opentelemetry-js/blob/e9d3c71918635d490b6a9ac9f8259265b38394d0/packages/sdk-metrics/src/export/PeriodicExportingMetricReader.ts#L164)). 
This if statement ensures that even during this shutdown process, these final metrics won't be sent if telemetry is disabled.